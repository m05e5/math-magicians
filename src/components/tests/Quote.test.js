import React from 'react';

import Quote from '../Quote';

const ReactTestRenderer = require('react-test-renderer');

it('Creates a snapshot of the quote page and it should match the previous one', () => {
  const tree = ReactTestRenderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
