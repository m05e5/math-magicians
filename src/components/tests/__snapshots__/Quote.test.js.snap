// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creates a snapshot of the quote page and it should match the previous one 1`] = `
<header>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    <span
      className="name"
    >
      Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers. —     Shakuntala Devi, Indian writer and mental calculator
    </span>
  </p>
</header>
`;
